html/css/styles.css

```diff
@@ -2163,4 +2163,20 @@ label {
 
 .device-icon-48h img {
   height: 48px;
+  max-width: 250px;
+}
+
+.device-title-bar-1 {
+  width: 25%;
+  max-width: 25%;
```

Comment:
I read up somewhere that its advisable to include it for "older, non compliant browsers" so I put in there for safety. It didn't make any difference to me, but wanted to try cover all bases. Want me to remove it?